src/ui/theme/theme.ts

```diff
@@ -121,4 +146,138 @@ export const theme = extendTheme({
     Menu: menuTheme,
     Link: linkTheme,
   },
+  textStyles: {
+    // Regular Variants
+    'text-regular-xs': {
+      fontSize: 'xs',
+      fontWeight: 'regular',
+      letterSpacing: '1%',
+      lineHeight: '16px',
```

Comment:
Let's avoid using units for line height. make it just a number which is a ratio of the font size. 

For example if the font size is 18px, and the intended line height is 24, then the line height is `1.33`, no units.

Even though this is less of a problem on modern browsers it still might cause collapsing the lines when users increase the font size.